src/lib/Theme.ts

```diff
@@ -243,12 +247,6 @@ export default class Theme {
         try {
             const currentContext = getActiveContext();
             const env = Env.getEnvValue();
```

Comment:
remove this if it is not being in use anywhere in this function @anuragpandey1115